src/constants.js

```diff
@@ -179,6 +179,10 @@ export const ERROR_CODE_ITEM_NAME_INVALID = 'item_name_invalid';
 export const ERROR_CODE_ITEM_NAME_TOO_LONG = 'item_name_too_long';
 export const ERROR_CODE_ITEM_NAME_IN_USE = 'item_name_in_use';
 export const ERROR_CODE_UPLOAD_FILE_LIMIT = 'upload_file_limit';
+export const ERROR_CODE_UPLOAD_CHILD_FOLDER_FAILED = 'child_folder_failed_upload';
+export const ERROR_CODE_STORAGE_LIMIT_EXCEEDED = 'storage_limit_exceeded';
```

Comment:
The new ones should all be ERROR_CODE_UPLOAD_*
So that it's a bit more descriptive where the error is likely being used and also helps in search.

Reply:
Will fix.